mffpy/xml_files.py

```diff
@@ -576,7 +573,18 @@ class Epochs(XML):
     }
 
     def __getitem__(self, n):
-        return self.epochs[n]
+        """If `n` is an int, interpret as index and return the
+        corresponding epoch in the list. If `n` is a str, return
+        a list of all epochs with name `n`, or the individual
+        epoch if only one epoch with name `n`."""
+        if isinstance(n, int):
+            return self.epochs[n]
+        elif isinstance(n, str):
+            matched = list(filter(lambda epoch: epoch.name == n, self.epochs))
+            return matched[0] if len(matched) == 1 else matched
```

Comment:
[required]  I think we should also do something if at the default case, such as:
```python
else:
    raise ValueError(f"unsupported argument type '{n}' :  {type(n)}")
```

Reply:
Good call.